src/controllers/userController.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const { UserService } = require('../services/usersService');
+const userService = new UserService();
+const getAllUsersController = (req, res) => {
```

Comment:
Same here and in all other cases

```suggestion
const getAllUsers = (req, res) => {
```